tests/test_filters.py

```diff
@@ -980,6 +980,21 @@ def test_filters_status_update(self, update):
         assert filters.StatusUpdate.WEB_APP_DATA.check_update(update)
         update.message.web_app_data = None
 
+        update.message.forum_topic_created = "topic"
+        assert filters.StatusUpdate.ALL.check_update(update)
+        assert filters.StatusUpdate.FORUM_TOPIC_CREATED.check_update(update)
+        update.message.forum_topic_created = None
+
+        update.message.forum_topic_closed = "topic"
```

Comment:
Not sure I understand this. Not least because `ForumTopicClosed` and `ForumTopicReopened` objects are supposed to be empty

Reply:
we only test if the filter works if `forum_topic_*` evaluates to `True`. This is one of the simplest ways to test it